test/libsolidity/semanticTests/externalContracts/_stringutils/stringutils.sol

```diff
@@ -53,7 +53,8 @@ library strings {
         }
 
         // Copy remaining bytes
-        uint mask = 256 ** (32 - len) - 1;
+        uint mask;
+        unchecked { mask = 256 ** (32 - len) - 1; }
```

Comment:
It causes a panic otherwise. I think overflow is expected?

Reply:
The loop condition shows that `len <= 32`. So there won't be a panic here.

Reply:
It can still end up with `len=32` which will do `256**32` and that exceeds `2**256-1`.

![Screenshot](https://user-images.githubusercontent.com/20340/116270862-61483100-a777-11eb-8020-a60a92142535.png)

Reply:
I you want to investigate, I suggest we remove the unchecked changes and the `benchmark` function from this PR so it can be merged, and then we keep the benchmark function open for investigation.

Reply:
Ah, of course `len =0` would lead to a panic.

Reply:
I'm fine with keeping the `unchecked`.

Reply:
Not sure which is nicer, adding an `if (len == 0) return;` is also an option, but I guess this unchecked option is more in-line with original intent and it tests our optimiser's capability of removing the no-op mload/mstores.